root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1579,6 +1579,26 @@ const testData = [
     expected_relationship_type: 'socialnetwork',
             expected_clean_url: 'https://www.instagram.com/yorickvannorden/',
   },
+  {
+                     input_url: 'https://www.instagram.com/stories/nathanwpylestrangeplanet/',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'socialnetwork',
+            expected_clean_url: 'https://www.instagram.com/nathanwpylestrangeplanet/',
+  },
```

Comment:
Since your patch adds validation, please test it by adding a property, to every Instagram test entry, as follows:

```suggestion
       only_valid_entity_types: ['artist'],
  },
```

Reply:
[ 'artist', 'event', 'label', 'place', 'series' ] in fact, but added :)